beta/lib/cli.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+'use strict';
+
+const list = require('cli-list');
+const meow = require('meow');
```

Comment:
Re-reading this — I do find I'm struggling a teeny bit to follow what the supported arguments to this are. 

Would it be worth using a somewhat more high-level CLI arguments parsing tool? Something like: https://www.npmjs.com/package/commander

Reply:
Discussed this with Spencer, we're going to stick with meow for the time being as its more flexible and light weight and commander seems to be more opinionated.